checkov/common/util/parser_utils.py

```diff
@@ -325,29 +328,29 @@ def get_current_module_index(full_path: str) -> Optional[int]:
 
 
 def is_nested(full_path: str) -> bool:
-    return '[' in full_path
+    return TERRAFORM_NESTED_MODULE_PATH_PREFIX in full_path
 
 
 def get_tf_definition_key(nested_module: str, module_name: str, module_index: Any, nested_key: str = '') -> str:
-    return f"{nested_module}[{module_name}#{module_index}{nested_key}]"
+    return f"{nested_module}{TERRAFORM_NESTED_MODULE_PATH_PREFIX}{module_name}#{module_index}{nested_key}{TERRAFORM_NESTED_MODULE_PATH_ENDING}"
 
 
 def get_tf_definition_key_from_module_dependency(path: str, module_dependency: str, module_dependency_num: str) -> str:
     if not module_dependency:
         return path
     if not is_nested(module_dependency):
-        return f"{path}[{module_dependency}#{module_dependency_num}]"
+        return f"{path}{TERRAFORM_NESTED_MODULE_PATH_PREFIX}{module_dependency}#{module_dependency_num}{TERRAFORM_NESTED_MODULE_PATH_ENDING}"
     module_index = get_current_module_index(module_dependency)
-    return f"{path}[{module_dependency[:module_index]}#{module_dependency_num}{module_dependency[module_index:]}]"
+    return f"{path}{TERRAFORM_NESTED_MODULE_PATH_PREFIX}{module_dependency[:module_index]}#{module_dependency_num}{module_dependency[module_index:]}{TERRAFORM_NESTED_MODULE_PATH_ENDING}"
 
 
 def get_module_from_full_path(file_path: str) -> Tuple[Optional[str], Optional[str]]:
     if not is_nested(file_path):
         return None, None
-    tmp_path = file_path[file_path.index('[') + 1: -1]
+    tmp_path = file_path[file_path.index(TERRAFORM_NESTED_MODULE_PATH_PREFIX) + 3: -3]
```

Comment:
Can we extract the used indexes (-3) into a constant? I see we're using it in `checkov/terraform/graph_builder/utils.py`  and `checkov/terraform/runner.py`